src/main.rs

```diff
@@ -1238,6 +1243,18 @@ mod cli {
         pub typescript: Option<TypeScriptScope>,
     }
 
+    #[derive(Parser, Debug, Clone)]
+    #[group(required = false, multiple = false)]
+    pub struct CScope {
+        /// Scope C code using a prepared query.
+        #[arg(long, env, verbatim_doc_comment, visible_alias = "c")]
```

Comment:
You can leave off the `visible_alias`, it's identical to the normal command. Doesn't get shorter than `c` 😄

Reply:
Oh, copypaste =) Deleted, thanks!